css/style.css

```diff
@@ -77,3 +122,4 @@ footer li {
      margin-right: 10px;
      display: inline-block;
  }
+>>>>>>> about-me
```

Comment:
Start over again, and never commit the conflict to the repo. Review your changes before commit